src/screens/ProfileScreen/elements/RenderItem.js

```diff
@@ -153,6 +155,9 @@ const Item = ({
         height={getHeightHeader()}
         showAnonymousOption={true}
         source={SOURCE_MY_PROFILE}
+        isFollow={item?.is_following_target}
+        onPressFollUnFoll={() => followUnfollow(item)}
+        disabledFollow={true}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isFollow` prop is now being set with `item?.is_following_target`, and a new `onPressFollUnFoll` prop is added which triggers the `followUnfollow` function. Also, `disabledFollow` is set to `true`. Ensure these changes align with your component's requirements and don't break existing functionality.
```diff
+ isFollow={item?.is_following_target}
+ onPressFollUnFoll={() => followUnfollow(item)}
+ disabledFollow={true}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->